user/common-build-problems.md

```diff
@@ -313,21 +337,21 @@ If you're having trouble tracking down the exact problem in a build it often hel
 ### Running a Container Based Docker Image Locally
 
 1. Download and install Docker:
-  * [Windows](https://docs.docker.com/docker-for-windows/)
```

Comment:
That markdown is not actually correct, new line and better indent means this is detected properly.